Clear extensions cache after discover/execution complete.

Faizan2304 opened the pull request.

Commit:
Clear cache after discover/execution complete.

Commit:
Sort usings

Faizan2304 commented:
@dotnet-bot test Windows_NT / Release Build

Faizan2304 commented:
@dotnet-bot test Windows_NT / Release Build

codito commented on src/Microsoft.TestPlatform.Client/Discovery/DiscoveryRequest.cs:
```diff
@@ -225,6 +226,7 @@ public void HandleDiscoveryComplete(long totalTests, IEnumerable<TestCase> lastC
                 }
                 finally
                 {
+                    TestPluginCache.Instance.ClearExtentions();
```
nit: API spelling is incorrect.

Faizan2304 replied:
Fixed!!

Commit:
Fix for issue:
1) https://github.com/Microsoft/vstest/issues/632
2) https://github.com/Microsoft/vstest/issues/844

Commit:
nitpick: Spelling

Commit:
Merge branch 'master' into clearCache

Faizan2304 commented:
@dotnet-bot test this please

Commit:
Addressed PR comment

Commit:
spelling correction

Commit:
nitpick: function name

codito commented on src/Microsoft.TestPlatform.Client/Discovery/DiscoveryRequest.cs:
```diff
@@ -8,6 +8,7 @@ namespace Microsoft.VisualStudio.TestPlatform.Client.Discovery
     using System.Linq;
     using System.Threading;
 
+    using Microsoft.VisualStudio.TestPlatform.Common.ExtensionFramework;
```
nit: is this used in the code? There is not new addition in the file.

codito commented:
@dotnet-bot test this please

Commit:
Removed unused usings

Faizan2304 committed:
Merge branch 'master' into clearCache

Faizan2304 commented:
@dotnet-bot test Windows_NT / Debug Build please
